website/docs/r/is_instance_volume_attachment.html.markdown

```diff
@@ -0,0 +1,138 @@
+---
+
+subcategory: "VPC infrastructure"
+layout: "ibm"
+page_title: "IBM : instance_volume_attachment"
+description: |-
+  Manages IBM Cloud infrastructure instance volume attachment.
+---
+
+# ibm_is_vpc_route
+Create, update, or delete a Volume attachment on an instance. For more information, about VPC virtual server instances, see [Managing virtual server instances](https://cloud.ibm.com/docs/vpc?topic=vpc-managing-virtual-server-instances).
+
+## Example usage (using capacity)
+
+```terraform
+resource "ibm_is_vpc" "testacc_vpc" {
+  name = "testvpc"
+}
+
+resource "ibm_is_subnet" "testacc_subnet" {
+  name            = "testsubnet"
+  vpc             = ibm_is_vpc.testacc_vpc.id
+  zone            = "us-south-2"
+  total_ipv4_address_count = 16
+}
+
+resource "ibm_is_ssh_key" "testacc_sshkey" {
+  name       = "testssh"
+  public_key = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCKVmnMOlHKcZK8tpt3MP1lqOLAcqcJzhsvJcjscgVERRN7/9484SOBJ3HSKxxNG5JN8owAjy5f9yYwcUg+JaUVuytn5Pv3aeYROHGGg+5G346xaq3DAwX6Y5ykr2fvjObgncQBnuU5KHWCECO/4h8uWuwh/kfniXPVjFToc+gnkqA+3RKpAecZhFXwfalQ9mMuYGFxn+fwn8cYEApsJbsEmb0iJwPiZ5hjFC8wREuiTlhPHDgkBLOiycd20op2nXzDbHfCHInquEe/gYxEitALONxm0swBOwJZwlTDOB7C6y2dzlrtxr1L59m7pCkWI4EtTRLvleehBoj3u7jB4usR"
+}
+
+resource "ibm_is_instance" "testacc_instance" {
+  name    = "testvsi1"
+  image   = "7eb4e35b-4257-56f8-d7da-326d85452591"
+  profile = "bc1-2x8"
+  primary_network_interface {
+    subnet     = ibm_is_subnet.testacc_subnet.id
+  }
+  vpc  = ibm_is_vpc.testacc_vpc.id
+  zone = "us-south-2"
+  keys = [ibm_is_ssh_key.testacc_sshkey.id]
+  network_interfaces {
+    subnet = ibm_is_subnet.testacc_subnet.id
+    name   = "eth1"
+  }
+}
+
+resource "ibm_is_instance_volume_attachment" "testacc_att1" {
+  instance = ibm_is_instance.testacc_instance.id
+
+  name = "test-vol-att-1"
+  profile = "general-purpose"
+  capacity = "20"
+  delete_volume_on_attachment_delete = true
+  delete_volume_on_instance_delete = true
+  volume_name = "testvol1"
+}
+
+```
+## Example usage (using existing volume)
+
+```terraform
+resource "ibm_is_volume" "testacc_vol" {
+  name    = "testvol2"
+  profile = "10iops-tier"
+  zone    = "us-south-2"
+}
+
+resource "ibm_is_instance_volume_attachment" "testacc_att2" {
+  instance = ibm_is_instance.testacc_instance.id
+
+  name = "test-col-att-2"
+  volume = ibm_is_volume.testacc_vol.id
+  delete_volume_on_attachment_delete = false
+  delete_volume_on_instance_delete = false
+}
+
+```
+## Example usage (restoring using snapshot)
+
+```terraform
+resource "ibm_is_instance_volume_attachment" "testacc_att3" {
+  instance = ibm_is_instance.testacc_instance.id
+
+  name = "test-col-att-3"
+  profile = "general-purpose"
+  snapshot = xxxx-xx-x-xxxxx
+  delete_volume_on_attachment_delete = true
+  delete_volume_on_instance_delete = true
+  volume_name = "testvol3"
+}
+
+```
+  
+
+## Argument reference
+Review the argument references that you can specify for your resource. 
+
```

Comment:
Arrange all arguments/attributes in Alphbetical order...and follow all new changes implemented recently...just check other VPC docs (nested bloack, alphabetical order, description...so on)